From 6cc507362f90d0c005a33b515620c8237dca0273 Mon Sep 17 00:00:00 2001
From: DaPigGuy <mcpepig123@gmail.com>
Date: Sat, 21 Jan 2023 20:59:28 -0800
Subject: [PATCH] hwdb: Prevent activation of airplane mode on HP ENVY x360

(cherry picked from commit 60eac27a0d52f017a57f07dbf8003aa9672bdec4)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 6824d5a393..47d4583ef2 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -609,6 +609,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPENVYx360Convertible13*:*
 
 # Spectre x360 16 2022
 evdev:name:Intel HID events:dmi:bvn*:bvr*:bd*:svnHP*:pn*HP[sS][pP][eE][cC][tT][rR][eE]*x3602-in-1*:*
+# ENVY x360
+evdev:name:Intel HID events:dmi:bvn*:bvr*:bd*:svnHP*:pnHPENVYx360Convertible*:*
  KEYBOARD_KEY_08=unknown                                #  Prevents random airplane mode activation
 
 # HP Elite x2 1013 G3
